logging/unit_tests/handlers/transports/test_background_thread.py

```diff
@@ -184,7 +184,7 @@ def batch(self):
 
 class _Client(object):
 
-    def __init__(self, project):
+    def __init__(self, project, _=None, __=None):
```

Comment:
I think you could just do `*args` here instead, but @dhermes probably has a better idea?